test/e2e-secondary-network/secondary_network_test.go

```diff
@@ -288,3 +267,87 @@ func TestVLANNetwork(t *testing.T) {
 	}
 	testSecondaryNetwork(t, networkTypeVLAN, pods)
 }
+
+func (data *testData) assignIP(clientset *kubernetes.Clientset) error {
+	e2eTestData := data.e2eTestData
+	namespace := e2eTestData.GetTestNamespace()
+
+	for _, testPod := range data.pods {
+		node, err := clientset.CoreV1().Nodes().Get(context.TODO(), testPod.nodeName, metav1.GetOptions{})
+		if err != nil {
+			return fmt.Errorf("error when getting the cluster Node %s: %v", testPod.nodeName, err)
+		}
+		eni, has := node.Labels["eni-id"]
```

Comment:
nit: can you use `exists` here instead of `has` to be consistent with the other map lookup below? Or you can use `ok` everywhere.